memcached.c

```diff
@@ -10043,6 +10044,16 @@ static void process_lqdetect_command(conn *c, token_t *tokens, size_t ntokens)
         }
         conn_set_state(c, conn_mwrite);
         c->msgcurr = 0;
+#endif
+#ifdef LQDETECT_SHOW
+        int size;
+        char *str = lqdetect_show(&size);
+        if (str) {
+            write_and_free(c, str, size);
+        } else {
+            out_string(c, "\tSERVER ERROR out of memory writing show response.\n");
```

Comment:
```suggestion
            out_string(c, "SERVER ERROR out of memory writing show response");
```
리뷰 단계에서 사소한 구현까지 짚어내다 보면 여러 사람의 시간을 많이 빼앗게 됩니다.

PR 등록 전에 충분히 확인하고 올려주기 바랍니다.

Reply:
조금 더 신경쓰도록 하겠습니다!